pt-BR/working_with_javascript_in_rails.md

```diff
@@ -436,20 +437,19 @@ The index view (`app/views/users/index.html.erb`) contains:
 <% end %>
 ```
 
-The `app/views/users/_user.html.erb` partial contains the following:
+A *partial* `app/views/users/_user.html.erb` contém o seguinte:
 
 ```erb
 <li><%= user.name %></li>
 ```
 
-The top portion of the index page displays the users. The bottom portion
-provides a form to create a new user.
+A parte de cima da página *index* mostra os usuários. A parte debaixo
+fornece o formulário para criar um novo usuário.
 
-The bottom form will call the `create` action on the `UsersController`. Because
-the form's remote option is set to true, the request will be posted to the
-`UsersController` as an Ajax request, looking for JavaScript. In order to
-serve that request, the `create` action of your controller would look like
-this:
+O formulário debaixo chamará a ação *`create`* no *`UsersController`*. por causa
```

Comment:
```suggestion
O formulário debaixo chamará a ação *`create`* no `UsersController`. por causa
```

Acho que como tá em código podemos tirar o itálico o que acha?

Reply:
boa! arrumei.